File: app/src/main/java/com/example/hades/androidpo/_1_render_op/startup/v1/RxLifeCycleSchedulerBinder.java
package com.example.hades.androidpo._1_render_op.startup.v1;

import io.reactivex.ObservableTransformer;

public interface RxLifeCycleSchedulerBinder extends RxLifeCycleBinder {
    <T> ObservableTransformer<T, T> bindWithScheduler();
}

File: app/src/main/java/com/example/hades/androidpo/_1_render_op/startup/v2/RxLifeCycleSchedulerBinder.kt
package com.example.hades.androidpo._1_render_op.startup.v2

import io.reactivex.ObservableTransformer

interface RxLifeCycleSchedulerBinder : RxLifeCycleBinder {
    fun <T> bindWithScheduler(): ObservableTransformer<T, T>?
}